src/framework/components/camera/component.js

```diff
@@ -289,6 +289,14 @@ class CameraComponent extends Component {
         }
     }
 
+    set renderSceneColorMap(value) {
+        if (value) {
+            this._colorRequestedBefore = true;
+            this.requestSceneColorMap(true);
+        } else if (this._colorRequestedBefore)
+            this.requestSceneColorMap(false);
```

Comment:
I think in the else branch we need to set _requestedBefore to false, so that multiple calls with false won't decrement counter more than once.

Reply:
So to be clear, we want this API to clamp the counter at 0? Because in that case, we could just use the counter to test if we should decrement or not.

Reply:
that's not the case. Any client can only cancel request made by themselves. They cannot cancel any request made by somebody else. We need a flag to know if we requested, and only cancel it one time.